.github/workflows/docs.yml

```diff
@@ -4,21 +4,52 @@ on:
   push:
     branches:
       - stable
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'Ref to deploy, defaults to `unstable`'
+        required: false
+        default: 'unstable'
+        type: string
 
 jobs:
   docs:
     runs-on: buildjet-4vcpu-ubuntu-2204
+    env:
+      DEPLOY_REF: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.ref || 'stable' }}
```

Comment:
A little strange that the default here is 'stable' but above, its 'unstable'. I'd say pick one or the other

Reply:
That was so that its deploying `stable` if running on_push and defaulting to `unstable` in the manual deploy option with workflow_dispatch.

I wanted to maintain the original behavior of docs deploying when merged to stable and assumed we would most likely be manually deploying from `unstable` after a docs PR but between releases that auto deploy.

Do you want me to convert the hardcoded branch to `github.ref_name` so that it will be more clear that its the value that `push` triggered the workflow?

Reply:
Oh I see, that makes sense! Looks good as is